app/controllers/application_controller.rb

```diff
@@ -11,4 +11,11 @@ class ApplicationController < ActionController::Base
   def current_user
     super || User.guest
   end
+
+  def append_info_to_payload(payload)
+    super
+    payload[:request_id] = request.env["action_dispatch.request_id"]
+    payload[:x_forwarded_for] = request.env["HTTP_X_FORWARDED_FOR"]
+    payload[:remote_addr] = request.env["REMOTE_ADDR"]
```

Comment:
@whereismyjetpack If I'm reading the docs right, isn't this is the same as what's happening in ` config/initializers/lograge.rb`? I prefer the initializer method as opposed to overriding a method, if we can do that.

Reply:
I _think_ this is an ordering thing, I don't think the request ID is available within the intializer, but it is here. I put all the items here so they'd at least be in the same place.

Reply:
The docs say "Alternatively, you can add a hook for accessing controller methods directly ..." Have you tried it without the change to the controller to see if it works?

Reply:
I thought I did. but i'll go back and try again!

Reply:
the controller method did the trick. I believe this to be resolved.